buildchow/templates/product-template.yaml

```diff
@@ -0,0 +1,163 @@
+          browser-url: https://github.com/jbdevstudio/jbdevstudio-product/
+          basedir: "sources"
+    nodes: "{nodes}"
+    jdk: jdk1.7
+
+    triggers:
+      - pollscm: "{scm-schedule}"    
+
+    builders:
+      - shell: |
+               if  [[ ! -d jbds-build-env ]]; then
+               ln -s /home/hudson/static_build_env/jbds jbds-build-env
+               fi
+               
+               #rm -rf ${{WORKSPACE}}/m2-repository
+               
+               # 2014/03/18: remove `-gs /home/hudson/.m2/settings.xml` from maven invocation below
+      - maven-target:
+          goals: "clean install ${{MAVEN_FLAGS}} -DTARGET_PLATFORM_VERSION=${{TARGET_PLATFORM_VERSION_MAXIMUM}} -Ppack200"
+          maven-version: maven-3.2.5
+          private-repository: true
+          java-opts: 
+            - "{jvm-memory}"
+          properties: 
+            - JOB_NAME=${{JOB_NAME}}
+            - BUILD_ID=${{BUILD_ID}}
+            - BUILD_NUMBER=${{BUILD_NUMBER}}
+            - maven.test.failure.ignore=true
+            - maven.test.error.ignore=true
+            - download.cache.directory={download-cache}
+            - targetPlatformGroup=jbdevstudio
+            - jbosstools-site=http://download.jboss.org/jbosstools/updates/nightly/core/${{jbosstools-site-stream}}/
+          pom: ${{WORKSPACE}}/product/site/pom.xml
+      - shell: |
+               # move the update site target folder temporarily
+               rm -fr ${{WORKSPACE}}/tmp-target
+               mv ${{WORKSPACE}}/sources/product/site/target ${{WORKSPACE}}/tmp-target
+               
+               # now rebuild the update site, without pack200
+      - maven-target:
+          goals: "clean install ${{MAVEN_FLAGS}} -DTARGET_PLATFORM_VERSION=${{TARGET_PLATFORM_VERSION_MAXIMUM}}"
+          maven-version: maven-3.1.1
+          private-repository: true
+          java-opts: 
+            - "{jvm-memory}"
+          properties: 
+            - JOB_NAME=${{JOB_NAME}}
+            - BUILD_ID=${{BUILD_ID}}
+            - BUILD_NUMBER=${{BUILD_NUMBER}}
+            - maven.test.failure.ignore=true
+            - maven.test.error.ignore=true
+            - download.cache.directory={download-cache}
+            - targetPlatformGroup=jbdevstudio
+            - jbosstools-site=http://download.jboss.org/jbosstools/updates/nightly/core/${{jbosstools-site-stream}}/
+          pom: ${{WORKSPACE}}/sources/product/site/pom.xml          
+      - maven-target:
+          goals: "org.apache.maven.plugins:maven-dependency-plugin:2.9:unpack"
+          maven-version: maven-3.x
+          private-repository: false
+          properties:
+            - artifact=org.jboss.tools.releng:jbosstools-releng-publish:4.3.0.Alpha1-SNAPSHOT:zip
+            - outputDirectory=${{WORKSPACE}}/sources
+            - trimVersion=true
+            - mdep.stripClassifier=true
+            - mdep.stripVersion=true
+      - shell: |
+               # move the non-pack200 site zip into the temporary folder
```

Comment:
yes, as noted in the pr comments and commits only the component jobs was updated to latest.